src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+Todo: add copy here
+
+### Giveaway scam (Double your ETH)
+
+One of the most common scams in cryptocurrency is the giveaway scam. The giveaway scam can take many forms, but the general premise is that if you send ETH to the provided wallet address, you will receive your ETH back but doubled. *For this reason, it is also known as the 2-for-1 scam.*
+
+These scams usually stipulate a limited time of opportunity to claim the giveaway to encourage poor decision-making and create a sense of false urgency.
+
+#### Twitter Hack
+
+A high-profile version of this occurred in July 2020, when the Twitter accounts of prominent celebrities and organisations got hacked. The hacker simultaneously posted a Bitcoin giveaway on the hacked accounts. Although the deceptive tweets were quickly noticed and deleted, the hackers still managed to get away with 11 bitcoin (or $500,000 as of September 2021)
+
+![A scam on Twitter](./twitterScam.png)
+
+#### Vitalik Giveaway
+
+The Vitalik giveaway is another common form the giveaway scam takes. The scammers will take a recorded video interview or conference talk given by Vitalik Buterin, co-founder of Ethereum, and livestream it on YouTube - making it appear as though Vitalik was giving a live video interview endorsing a cryptocurrency giveaway.
+
+Vitalik is used most often in this scam, but many other prominent people involved in crypto are also used (e.g. Elon Musk or Charles Hoskinson). Including a well-known person gives the scammers livestream a sense of legitimacy (this looks sketchy, but Vitalik is involved, so it must be ok!).
+
+**Giveaways are always scams. If you send your funds to these accounts, you will lose them forever.**
+
+![A scam on YouTube](./youtubeScam.png)
+
+### Support Scams
+
+Cryptocurrency is a relatively young and misunderstood technology. A common scam that takes advantage of this is the support scam, where scammers will impersonate support personnel for popular wallets, exchanges, or blockchains.
+
+Much of the discussion about Ethereum happens on Discord. Support scammers will commonly find their target by searching for support questions in public discord channels and then sending the enquirer a private message offering support. By building trust, support scammers try to trick you into revealing your private keys or sending your funds to their wallets.
+
+![A support scam on Discord](./supportScam.png)
+
+As a general rule, staff will never communicate with you through private, unofficial channels. Some simple things to keep in mind when dealing with support:
+
+- Never share your private keys, seed phrases or passwords
+- Never allow anyone remote access into your computer
+- Never communicate in unofficial private channels
+
+<InfoBanner emoji=":lock:">
+  <div>
+    Beware: although support-style scams commonly happen on Discord, they can also be prevalent on any chat applications where crypto discussion happens, including email.
+  </div>
+</InfoBanner>
+
+### Phishing Scams
```

Comment:
Good resource to potentially mention here: https://support.mycrypto.com/staying-safe/mycrypto-protips-how-not-to-get-scammed-during-ico